cinema/migrations/0001_initial.py

```diff
@@ -1,5 +1,6 @@
-# Generated by Django 4.0.4 on 2022-05-25 13:21
```

Comment:
Please do not modify existing migration file.
You should create a new one instead